extension/lib/PrefUtils.jsm

```diff
@@ -0,0 +1,15 @@
+const { utils: Cu } = Components;
+Cu.import("resource://gre/modules/Services.jsm");
+
+
+const PrefUtils = {
+  setInt64Pref(name, value) {
```

Comment:
There isn't anything here that makes this 64bit. A better name might be `setLongPref` or something?